src/utils/validateTrimmedString.ts

```diff
@@ -1,10 +1,41 @@
-// makes sure the string passed into the function isn't an empty string
-export function validateTrimmedString(input: string) {
-	const trimmedInput = input.trim();
+import { ListItem } from "../api";
 
+// Validates the item name input for a shopping list
+export function validateItemName(
```

Comment:
praise: amazing at extending functionality and bringing the new logic and the current logic together to make an awesome new piece of code!